schema.graphql

```diff
@@ -10333,31 +10339,6 @@ type Enterprise implements Node {
   """
   url: URI!
 
-  """
-  A list of user accounts on this enterprise.
-  """
-  userAccounts(
-    """
-    Returns the elements in the list that come after the specified cursor.
-    """
-    after: String
-
-    """
-    Returns the elements in the list that come before the specified cursor.
-    """
-    before: String
-
-    """
-    Returns the first _n_ elements from the list.
-    """
-    first: Int
-
-    """
-    Returns the last _n_ elements from the list.
-    """
-    last: Int
-  ): EnterpriseUserAccountConnection! @deprecated(reason: "The `Enterprise.userAccounts` field is being removed. Use the `Enterprise.members` field instead. Removal on 2022-07-01 UTC.")
```

Comment:
**⚠️ This is a breaking change**

`Enterprise.userAccounts` was deprecated in favour of `Enterprise.members`, with notice given that it would disappear around `2022-07-01`. According to our logs, there were only single-digit numbers of people still using it.

**This change is safe to merge.**